nickytonline/astro-partykit-starter
# feat: adding playwright testing to starter kit

## Description

This PR does the following: 

- [x] installing Playwright to starter kit
- [x] add test for joining a room with username
- [x] add test for joining as anonymous user to the default room
- [x] add test for the modal
- [x] add test for the sending messages
- [x] updates the `README` to mention testing with playwright

## What type of PR is this? (check all applicable)


## Related Tickets & Documents
closes #53 

## Mobile & Desktop Screenshots/Recordings

<img width="1124" alt="Screenshot 2024-02-03 at 12 39 35 PM" src="https://github.com/nickytonline/astro-partykit-starter/assets/67210629/130c027a-bd45-424c-bc72-c8e17b576372">



## Steps to QA

1. pull down this branch
2. install dependencies 
3. run `npx playwright test` in the project directory
4. run `npx playwright show-report` to see results on `http://localhost:9323`



## Added to documentation?


## [optional] Are there any post-deployment tasks we need to perform?



## [optional] What gif best describes this PR or how it makes you feel?
